### PR TITLE
Add instructions for using Chrome remote debugger

### LINKED ISSUE #4626: Enable Browser Dev tools Debug console for the `main` builds
### Problem Description

At present, getting into the Electron debug mode (browser dev tools) is possible only in the dev mode (`npm run dev`) and may be PR builds?

As `npm run dev` setup  sometimes doesn't work on some machines, It will be very helpful if we can enable the debug mode behind an environment variable (ex: RD_DEBUG_MODE) for the `main` builds too. 

### Proposed Solution

Enable the debug mode behind an environment variable (ex: RD_DEBUG_MODE) for the `main` builds too. 

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by rak-phillip**
This adds instructions for running production builds of Rancher Desktop with the Chrome remote debugger attached. 

This is a follow up from our conversation in #4626

closes #4626

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -195,6 +195,39 @@ npm run package --publish=never
 
 The build output goes to `dist/`.
 
+### Debugging builds with the Chrome remote debugger
+
+The Chrome remote debugger allows you to debug Electron apps using Chrome Developer Tools. You can use it to access log messages that might output to the developer console of the renderer process. This is especially helpful for getting additional debug information in production builds of Rancher Desktop.
```
Comment (gunamata) [inline README.md:200]: Even though it's not difficult to find out the folder where Rancher Desktop is installed, it helps if we provide that information here.

**Review by gunamata — COMMENTED** [review]
LGTM, Thank you. The steps worked as expected on my Windows machine. I just have one suggestion about providing the installation directory information.

File: README.md (hunk 1 above)
Comment (rak-phillip) [reply README.md:200]: I updated the PowerShell example to change the directory to the Rancher Desktop install location before starting with remote debugging enabled.

**Review by gunamata — APPROVED** [review]
LGTM, Thank you.

**Pull request merged**